integration/spark-common-cluster-test/src/test/scala/org/apache/carbondata/cluster/sdv/generated/SetParameterTestCase.scala

```diff
@@ -231,6 +231,26 @@ class SetParameterTestCase extends QueryTest with BeforeAndAfterAll {
     assert(getLogFileCount("default", "carbon_table_load", "0") >= 1)
   }
 
+  test("TC_013-test set dynamic properties") {
+    sql("SET carbon.number.of.cores.while.loading=8")
```

Comment:
you need to reset also, else it affect the other test cases

Reply:
modified